servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRest.java

```diff
@@ -55,26 +51,22 @@ public abstract class AbstractRest {
     Locale.setDefault(Locale.ENGLISH);
   }
 
-  protected void init(URI uri) {
-    NessieApiV1 api = HttpClientBuilder.builder().withUri(uri).build(NessieApiV1.class);
-
-    ObjectMapper mapper =
-        new ObjectMapper()
-            .enable(SerializationFeature.INDENT_OUTPUT)
-            .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);
-    HttpClient.Builder httpClient = HttpClient.builder().setBaseUri(uri).setObjectMapper(mapper);
-    httpClient.addResponseFilter(new NessieHttpResponseFilter(mapper));
-
-    init(api, httpClient);
+  protected void initApi(URI nessieApiUri) {
+    NessieApiV1 api = HttpClientBuilder.builder().withUri(nessieApiUri).build(NessieApiV1.class);
+    initApi(api);
   }
 
-  protected void init(NessieApiV1 api, @Nullable HttpClient.Builder httpClient) {
```

Comment:
`httpClient` param is ignored here (it used to be stored into a field previously)